language-support/qute/qute.tmLanguage.json

```diff
@@ -133,6 +139,70 @@
         }
       }
     },
+    "section_start_if_tag": {
+      "begin": "({)(#)((if)\\b)",
+      "end": "(\\/)?((?<![\\\\])})",
+      "beginCaptures": {
+        "1": {
+          "name": "support.constant.handlebars"
+        },
+        "2": {
+          "name": "keyword.control"
+        },
+        "3": {
+          "name": "keyword.control"
+        }
+      },
+      "endCaptures": {
+        "1": {
+          "name": "keyword.control"
+        },
+        "2": {
+          "name": "support.constant.handlebars"
+        }
+      },
+      "patterns": [
+        {
+          "match": "(gt|ge|lt|le|eq|is|ne|and|or)",
+          "name": "keyword.control.qute"
+        },
+        {
```

Comment:
Please defines in a new match the other operator defined in https://quarkus.io/guides/qute-reference#if_section (like  `!=`)